laser_encoders/__main__.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+
+from laser_encoders.download_models import download_models
+
+
+def main():
```

Comment:
Why did you move the code from `download_models` here?

Reply:
In other to be able to this
`python -m laser_encoders download_models`

with the way it was structured before, you had to run it like this
`python -m laser_encoders.download_models`

Reply:
The original way seems find to me?

Reply:
I would suggest simply reverting.

Reply:
Alright.